app.js

```diff
@@ -208,23 +207,42 @@ const resCacher = {
   setContentType: 'application/json'
 }
 
+const startupTime = Date.now()
+const checkIfProcessIsTooOld = function(cont) {
+  if (typeof Settings.processLifespanLimitMs === 'string') {
+    Settings.processLifespanLimitMs = parseInt(Settings.processLifespanLimitMs)
+    Settings.processLifespanLimitMs +=
+      Settings.processLifespanLimitMs * (Math.random() / 10)
+  }
```

Comment:
This initial configuration of  `Settings.processLifespanLimitMs` only needs to run once at startup so it could be moved outside the function.

It might be simpler to compute an initial `shutdownTime` at the start, then you just need to compare it with Date.now().

Is it worth logging something at startup to show the lifespan parameter?